packages/airnode-admin/src/cli.ts

```diff
@@ -520,6 +520,13 @@ yargs
       console.log(`Airnode address: ${airnodeAddress}`);
     }
   )
+  .example([
+    ['$0 derive-airnode-address --airnode-mnemonic "test test test test test test test test test test test junk"\n'],
+    [
+      '$0 derive-sponsor-wallet-address --airnode-xpub xpub6Ce9NcJvTk36xtLSrJLZqE7wtgA5deCeYs7rSQtreh4cj6\nByPtrg9sD7V2FNFLPnf8heNP3FGkeV9qwfzvZNSd54JoNXVsXFYSYwHsnJxqP --airnode-address 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 --sponsor-address 0x70997970c51812dc3a010c7d01b50e0d17dc79c8\n',
```

Comment:
```suggestion
      '$0 derive-sponsor-wallet-address --airnode-xpub xpub6Ce9NcJvTk36xtLSrJLZqE7wtgA5deCeYs7rSQtreh4cj6ByPtrg9sD7V2FNFLPnf8heNP3FGkeV9qwfzvZNSd54JoNXVsXFYSYwHsnJxqP --airnode-address 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 --sponsor-address 0x70997970c51812dc3a010c7d01b50e0d17dc79c8\n',
```

otherwise the command doesn't work.